source-api/consumer-proguard.pro

```diff
@@ -2,4 +2,8 @@
 -keep class eu.kanade.tachiyomi.source.online.** { public protected *; }
 -keep class eu.kanade.tachiyomi.source.** extends eu.kanade.tachiyomi.source.Source { public protected *; }
 
+-keep class eu.kanade.tachiyomi.animesource.model.** { public protected *; }
```

Comment:
Oh wow I didn't know that module also had a proguard file bruh

Reply:
me neither, I just did a search for proguard in the files and found it